### ISSUE TITLE
Transparent name background & remove name colorize

### ISSUE THREAD

**Issue by novikmisha** [obble/modui_classic#46]
Can we have an option to remove name background texture? And also stop colorizing name to leave it like (1, 0.8, 0) (also like an option)? For me, with this options unitframes become amazing. I did it in my fork but you make a lot of stuff today so I dont have time to rebase fork. Can provide you a screenshot of how it will look after those changes.

**Comment by warsken** [comment]
Yes! And second that - the target frame name background aswell. Like this 
![Screen Shot 2019-09-03 at 14 23 58](https://user-images.githubusercontent.com/54398264/64169455-8f685700-ce56-11e9-942a-08f05d5fd836.png)

**Comment by obble** [comment]
Good idea - I'd like to generally improve the amount of customisation the user has access to on unitframes for things like this. I'm planning on working on it in the next few days before I go away and have no access to a computer across next week :)

**Comment by AmplusNLD** [comment]
I also like the idea of having the HP bar class colored. I used to have that on retail aswell. And have it colored to the aggro state for general mobs. Like red or yellow.
